openpype/plugins/publish/collect_otio_review.py

```diff
@@ -87,7 +87,9 @@ def process(self, instance):
                 otio_review_clips.append(otio_gap)
 
         if otio_review_clips:
-            instance.data["label"] += " (review)"
+            # add review track to instance and change label to reflect it
+            label = instance.data.get("label", instance.data["subset"])
+            instance.data["label"] = label + " (review)"
```

Comment:
Isn't the default fallback for `label` the `instance.data["name"]` entry instead of subset?